README.md

```diff
@@ -7,7 +7,7 @@
 
 Makes your voice assistant laugh like a maniac
 
-![A ghostly face, laughing evilly.](./sshot.png)
+![A ghostly face, laughing evilly.](./ui/images/2.jpg)
```

Comment:
images should be moved to "gui/all", this ensures in docker setups they are available across containers as the "gui" folder gets copied to a shared volume

Reply:
similarly, in the code we should no longer pass absolute paths to allow the cached path to resolve instead

` self.gui.show_image(join(dirname(__file__), "ui", "images", str(pic) + ".jpg"))` -> ` self.gui.show_image(str(pic) + ".jpg"))`

Reply:
Do we have this documented somewhere? Wasn't aware this was an option

Reply:
Looks like this is in #38